fbrp/src/fbrp/cmd/logs.py

```diff
@@ -8,11 +9,22 @@
 
 
 @click.command()
-@click.argument("procs", nargs=-1)
+@click.argument(
+    "procs",
+    nargs=-1,
+    shell_complete=_autocomplete.conditional(
```

Comment:
Add a comment explaining the logic here -- requires some time thinking about the higher-order function to parse this.

Reply:
Added a comment